.changeset/chatty-experts-smell.md

```diff
@@ -0,0 +1,20 @@
+---
+"astro": minor
+---
+
+The CSRF protection check is out of experimental, and its configuration is changed:
```

Comment:
@ematipico if you get a chance to beef this up a bit more before I get here, great! You can use the `domains` PR as an example of ideally what goes here! If not, I will make this match before it's merged. (Just want to spend my time in the docs PRs first!)

Reply:
Done